input/resources/au-erequesting-server.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+  <id value="au-erequesting-server"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://hl7.org.au/fhir/ereq/CapabilityStatement/au-erequesting-server"/>
+  <name value="AUeRequestingServerCapabilityStatement"/>
+  <title value="AU eRequesting Server CapabilityStatement"/>
+  <status value="draft"/>
+  <description value="This CapabilityStatement describes the basic rules for the [AU eRequesting Server actor](ActorDefinition-au-erequesting-actor-server.html) that is  is a system that accepts diagnostic service requests and makes diagnostic service requests accessible. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU eRequesting servers are defined in this CapabilityStatement."/>
+  <kind value="requirements"/>
+  <fhirVersion value="4.0.1"/>
+  <format value="json">
+    <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+      <valueCode value="MAY"/>
+    </extension>
+  </format>
+  <format value="xml">
+    <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+      <valueCode value="MAY"/>
+    </extension>
+  </format>
+  <patchFormat value="application/json-patch+json">
+    <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+      <valueCode value="MAY"/>
+    </extension>
+  </patchFormat>
+  <implementationGuide value="http://hl7.org.au/fhir/ImplementationGuide/hl7.fhir.au.base">
+    <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+      <valueCode value="SHALL"/>
+    </extension>
+  </implementationGuide>
+  <implementationGuide value="http://hl7.org.au/fhir/core/ImplementationGuide/hl7.fhir.au.core">
+    <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+      <valueCode value="SHALL"/>
+    </extension>
+  </implementationGuide>
+  <rest>
+    <mode value="server"/>
+    <documentation value="TBD"/>
+    <security>
+      <description value="TBD"/>
```

Comment:
1. For rest.documentation either remove that field or replace the TBD with what is currently stateable e.g. SHALL support ServiceRequest.
2. For security.description since this description field is displayed to reader, it needs to be "None defined." rather than TBD since there is no forthcoming change expected to this prior to ballot.

Reply:
1. is not sufficient, my comment non-exhaustive, and also not intended to be an exact quote. more please with better wording and formatting. Look at AU Core, US Core, and IPA for the text expectations.
2. Confirmed fix.

Reply:
1. Addressed.